Index: configure.ac
===================================================================
--- configure.ac	(revision 5594)
+++ configure.ac	(working copy)
@@ -118,10 +118,14 @@
     [AC_MSG_WARN([Building the runtime to use libce with lib32 or lib64 is unsupported.])])])
 
 AS_CASE([$enable_w32api],
-  [yes],[
+  [yes],[AS_CASE([$host_cpu],
+	  [x86_64],[
+	    AC_SUBST([LIB64SUFFIXDIR],[lib/w32api])
+	    AC_SUBST([LIB32SUFFIXDIR],[lib32/w32api])],
+	  [i*86],[
 	    AC_SUBST([LIB64SUFFIXDIR],[lib64/w32api])
-	    AC_SUBST([LIB32SUFFIXDIR],[lib/w32api])
-	],
+	    AC_SUBST([LIB32SUFFIXDIR],[lib/w32api])]
+	)],
   [AS_CASE([$host_cpu],
 	  [x86_64],[
 	    AC_SUBST([LIB64SUFFIXDIR],[lib])
